docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -1,105 +1,192 @@
 
-Navigate to the latest GitHub [release](https://github.com/elastic/detection-rules/releases)), with the tag `ML-HostRiskScore-YYYMMDD-N`. From under `Assets`, download the zip file named `ML-HostRiskScore-YYYMMDD-N.zip` and unzip it. New releases may contain updated artifacts. 
+ 1. [Obtain artifacts](#obtain-artifacts) 
+ 2. [Upload scripts](#upload-scripts)
+ 3. [Upload ingest pipeline](#upload-ingest-pipeline)
+ 4. [Upload and start the `pivot` transform](#upload-start-pivot)
+ 5. [Create the Host Risk Score index](#host-risk-index)
+ 6. [Upload and start the `latest` transform](#upload-start-latest)
+ 7. [Import dashboards](#import-dashboards)
+ 8. [Enable Kibana features](#enable-kibana)
 
-#### 2. Modify artifacts to reflect Kibana space
+<h3 id="modify-artifacts">1. Obtain artifacts</h3>
 
-For security reasons, we require that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `ml_hostriskscore_generate_scripts.py` script in the directory that is prefixed with `ML-HostRiskScore` with your Kibana space as the argument.
+The Host Risk Score functionality is space aware for privacy. Downloaded artifacts must be modified with the desired space before they can be used.
 
-Eg:
+ - Download the latest release [bundle](https://github.com/elastic/detection-rules/releases) with the tag `ML-HostRiskScore-YYYMMDD-N`.
+ - Unzip the contents of `ML-HostRiskScore-YYYMMDD-N.zip`.
+ - Run `ml_hostriskscore_generate_scripts.py` script in the unzipped directory with your Kibana space as the argument.
+<div style="margin-left: 40px">   
+<i>Example of modifying artifacts for default space</i>
+   <pre style="margin-top:-2px"><code>python ml_hostriskscore_generate_scripts.py --space default
+</code></pre></div>
 
-```
-python ml_hostriskscore_generate_scripts.py --space default
-```
+ - Find a new folder named after your space in the unzipped directory. **You will be using the scripts within this directory for the next steps.**
 
-This will create a folder named `<your-space-name>`, consisting of all the required artifacts, modified for your Kibana space. **For the steps that follow, please use the scripts from this folder.**
+<h3 id="upload-scripts">2. Upload scripts</h3>
 
-#### 3. Navigate to the Dev Tools console in Kibana
+- Navigate to `Management / Dev Tools` in Kibana.
+- Upload the contents of `ml_hostriskscore_levels_script.json`, `ml_hostriskscore_map_script.json` and `ml_hostriskscore_reduce_script.json` using the Script API with the following syntax.
+- Ensure that your space name (such as `default`) replaces `<your-space-name>` in the script names below.
 
-You will now upload all the Host Risk Score artifacts to Kibana. Remember to suffix the names of all the scripts, ingest pipelines, transforms etc. with your Kibana space, while uploading them. 
-#### 4. Uploading required scripts
+<div style="margin-left: 40px">   
+<i>uploading scripts</i>
+   <pre style="margin-top:-2px"><code>
```

Comment:
Did something generate this markup, or were these tweaked manually? A `style` attribute like this seems unnecessary for documentation.